uiServices/src/configServices.go

```diff
@@ -28,15 +28,21 @@ func getConfigHeader(req *http.Request) string {
 }
 
 func ValidateFileNameAndHeader(rw http.ResponseWriter, req *http.Request, header, name string) bool {
+	return IsNameValid(name, rw) && IsHeaderValid(header, rw)
```

Comment:
see my comment from yesterday about removing the responsewriter functionality also